templates/default/sql/dml/1.3.0.sql.erb

```diff
@@ -11,3 +11,10 @@ DELETE FROM `hopsworks`.`variables` WHERE `id`='hiveserver_ext_hostname';
 DELETE FROM `hopsworks`.`variables` WHERE `id`='hiveserver_ssl_hostname';
 REPLACE INTO `hopsworks`.`variables` VALUES ("hopsworks_enterprise", "<%= node['install']['enterprise']['install']%>");
 REPLACE INTO `hopsworks`.`variables` VALUES ("kubernetes_installed", "<%= node['install']['kubernetes'] %>");
+INSERT INTO `hopsworks`.`variables` VALUES ("yarn_runtime", "docker");
+INSERT INTO `hopsworks`.`variables` VALUES ("docker_mounts", "<%= node['hops']['docker']['mounts'] %>");
+REPLACE INTO `hopsworks`.`variables` VALUES ("jupyter_user", "<%= node['hops']['yarnapp']['user'] %>");
+REPLACE INTO `hopsworks`.`variables` VALUES ("jupyter_group", "<%= node['hops']['group'] %>");
+REPLACE INTO variables VALUES ("anaconda_dir", "/srv/hops/anaconda");
+INSERT INTO `hopsworks`.`variables` VALUES ("docker_registry_name", "<%= @registry_host %>");
+INSERT INTO `hopsworks`.`variables` VALUES ("docker_registry_port", "<%= node['hops']['docker']['registry']['port'] %>");
```

Comment:
These should be discoverable from consul as well (cc. @kouzant )

Reply:
Consul did not exist when I started this work. Can we please get this in and then improve it in future jiras with the help of the people that know the code?